### PR TITLE
#4919 Update search box style / size

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/4919

Screen Shot 2020-01-10 at 1 47 18 AM

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 💙
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/w9jh5ew**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/filter-sort/components/filter-search.styles.less [hunk 1]
```diff
@@ -11,11 +11,13 @@
...
   .text-16;
+  height: @size-40;
 
   align-items: center;
   background: @color-dark-grey;
-  border: none;
-  border-radius: @size-6;
+  border: 1px solid @color-secondary-action;
+  box-sizing: border-box;
+  border-radius: @size-3;
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/filter-sort/components/filter-search.styles.less]: we actually have a `@border-radius-default` variable that we should use here (it's also 3px) just for future mass adjustments if we need to do them.

**Review by JohnDanz — CHANGES_REQUESTED** [review]
Aside from that one minor styling request this looks good, thanks :)

File: packages/augur-ui/src/modules/filter-sort/components/filter-search.styles.less (hunk 1 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/filter-sort/components/filter-search.styles.less]: Updated. Thank you.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**